deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
```
function create_alias() {
    local output=""
    output=$(curl -s --cacert $CA --key $KEY --cert $CERT -XPOST "https://$es_host:$es_port/_aliases" -d "{ \"actions\": [ { \"add\": { \"index\": \"${1}.*\", \"alias\": \"${1}.${2}.reference\"}} ] }")

    echo Migration for project $1: $output
}
...
    uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
    create_alias project.$index $uid
```
There are two cases here:
- upgrade from 1.old to 1.4 - this means they will need to move index `$projname.YYYY.MM.DD` to `project.$projname.$projuuid.YYYY.MM.DD`
- upgrade from 1.newer to 1.4 - this means they will need to move index `$projname.$projuuid.YYYY.MM.DD` to `project.$projname.$projuuid.YYYY.MM.DD`

Do we care about either of these cases?  If so, do we need to handle them with uuid_migrate.sh, or with index template alias definitions, or ???